main/src/main/java/cgeo/geocaching/activity/AbstractActionBarActivity.java

```diff
@@ -1,20 +1,62 @@
 package cgeo.geocaching.activity;
 
+import cgeo.geocaching.R;
+import cgeo.geocaching.ui.ViewUtils;
+import cgeo.geocaching.utils.Log;
+
 import android.os.Bundle;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.ActionBar;
+import androidx.core.graphics.Insets;
+import androidx.core.view.OnApplyWindowInsetsListener;
+import androidx.core.view.ViewCompat;
+import androidx.core.view.WindowCompat;
+import androidx.core.view.WindowInsetsCompat;
+import androidx.core.view.WindowInsetsControllerCompat;
 
 /**
  * Classes actually having an ActionBar (as opposed to the Dialog activities)
  */
 public class AbstractActionBarActivity extends AbstractActivity {
 
+    protected int calculateInsets = WindowInsetsCompat.Type.statusBars() | WindowInsetsCompat.Type.displayCutout();
+    boolean skipActionBarInsetCalculation = false; // for edge to edge configuration, see onCreate()
```

Comment:
I perrsonally don't like the usage of attributes to transport config info from subclass to superclass. This requires the subclass to know when it needs to set the attribute (before first usage). Maybe using methods instead?

Reply:
changed to `configureEdge2Edge()` method call. Still needs to be called before `onCreate()`, though.